src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+
+type Props = {
+  filterTodos: (value: string) => void;
+  countActive: number;
+  filterValue: string;
+};
+
+export const Footer: React.FC<Props> = (
+  { filterTodos, countActive, filterValue },
+) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${countActive} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          data-cy="FilterLinkAll"
+          href="#/"
+          className={`filter__link ${filterValue === 'All' && 'selected'}`}
+          onClick={(e) => {
+            filterTodos((e.target as HTMLAnchorElement).text);
+          }}
```

Comment:
Move it to a separate function and use it where needed. This function is repeated more than once